google/cloud/forseti/services/server/service.py

```diff
@@ -0,0 +1,155 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Server Config gRPC service. """
+
+import json
+import logging
+
+from google.cloud.forseti.services.server import server_pb2
+from google.cloud.forseti.services.server import server_pb2_grpc
+from google.cloud.forseti.common.util import logger
+
+
+LOGGER = logger.get_logger(__name__)
+
+
+class GrpcServiceConfig(server_pb2_grpc.ServerServicer):
+    """Service Config gRPC implementation."""
+
+    def __init__(self, service_config):
+        """Init.
+
+        Args:
+            service_config (ServiceConfig): Forseti server configs.
+        """
+        self.service_config = service_config
+
+    def Ping(self, request, _):
+        """Provides the capability to check for service availability.
+
+        Args:
+            request (PingRequest): The ping request.
+            _ (object): Context of the request.
+
+        Returns:
+            PingReply: The response to the ping request.
+        """
+
+        return server_pb2.PingReply(data=request.data)
+
+    def GetLogLevel(self, request, _):
+        """Get Log level.
+
+        Args:
+            request (PingRequest): The ping request.
+            _ (object): Context of the request.
+
+        Returns:
+            GetLogLevelReply: The GetLogLevelReply grpc object.
+        """
+        del request
+
+        log_level = logging.getLevelName(LOGGER.getEffectiveLevel())
+
+        LOGGER.info('Retrieving log level, log_level = %s',
+                    log_level)
+
+        return server_pb2.GetLogLevelReply(log_level=log_level)
+
+    def SetLogLevel(self, request, _):
+        """Set log level.
+
+        Args:
+            request (SetLogLevelRequest): The grpc request object.
+            _ (object): Context of the request.
+
+        Returns:
+            SetLogLevelReply: The SetLogLevelReply grpc object.
+        """
+        err_msg = ''
+
+        try:
+            LOGGER.info('Setting log level, log_level = %s',
+                        request.log_level)
+            logger.set_logger_level(logger.LOGLEVELS[request.log_level])
+        except Exception as e:  # pylint: disable=broad-except
+            err_msg = e.message
+
+        success = not err_msg
+
+        return server_pb2.SetLogLevelReply(success=success,
+                                           error_message=err_msg)
+
+    def ReloadServerConfiguration(self, request, _):
+        """Reload Server Configuration.
+
+        Args:
+            request (ReloadServerConfigurationRequest): The grpc request object.
+            _ (object): Context of the request.
+
+        Returns:
+            ReloadServerConfigurationReply: The ReloadConfigurationReply grpc object.
+        """
+
+        LOGGER.info('Reloading server configurations')
+        success, err_msg = self.service_config.update_configuration(
```

Comment:
Instead of `success`, this should be more clear as `is_success`, so that we can avoid something where `success = False`